component/appcat_apiserver.jsonnet

```
@@ -15,6 +15,13 @@ local namespace = loadManifest('namespace.yaml') {
   },
 };
 
+local envs = loadManifest('apiserver-envs.yaml') {
+  data+: {
+    APPCAT_HANDLER_ENABLED: apiserverParams.handlers.APPCAT_HANDLER_ENABLED,
+    VSHN_POSTGRES_BACKUP_HANDLER_ENABLED: apiserverParams.handlers.VSHN_POSTGRES_BACKUP_HANDLER_ENABLED,
+  },
```

What we could actually do here is instead of adding a specific `handlers` parameter to add a generic `env` parameter that would allow us to set or add any environment variables in the component without having to change any jsonnet.

```suggestion
  data+: apiserverParams.env,
```